skiko/src/jvmMain/cpp/common/jni_helpers.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+template<typename T>
+T fromJavaPointer(jlong ptr) { return reinterpret_cast<T>(static_cast<uintptr_t>(ptr)); }
```

Comment:
You could also make those inline.

Reply:
And return T*

Reply:
> You could also make those inline.
Done

> And return T*

see https://github.com/JetBrains/skiko/pull/62#discussion_r569530245